cameraview/src/main/java/com/otaliastudios/cameraview/Camera1.java

```diff
@@ -366,6 +369,20 @@ private boolean mergeHdr(Camera.Parameters params, Hdr oldHdr) {
         return false;
     }
 
+    @TargetApi(17)
+    private boolean mergePlaySound(boolean oldPlaySound) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+            Camera.CameraInfo info = new Camera.CameraInfo();
+            Camera.getCameraInfo(mCameraId, info);
+            if (info.canDisableShutterSound && isCameraAvailable()) {
```

Comment:
Can you remove isCameraAvailable here? it's already managed by the schedule() method